tests/ctst/common/common.ts

```diff
@@ -237,7 +269,8 @@ When('i restore object {string} for {int} days', async function (this: Zenko, ob
 // wait for object to transition to a location or get restored from it
 Then('object {string} should be {string} and have the storage class {string}', { timeout: 130000 },
     async function (this: Zenko, objectName: string, objectTransitionStatus: string, storageClass: string) {
-        const objName = objectName ||  this.getSaved<string>('objectName');
+        const objName =
+            getObjectNameWithBackendFlakyness.call(this, objectName) || this.getSaved<string>('objectName');
```

Comment:
```suggestion
        const objName =
            getObjectNameWithBackendFlakyness.call(this, objectName);
```

the `||` is useless as `getObjectNameWithBackendFlakyness` always returns a value ?

Reply:
It returns undefined in case we dont specify the object name, that's why there was ` this.getSaved<string>('objectName');` before my modification

Reply:
It returns undefined only when `backendFlakyness` and `backendFlakynessRetryNumber` are not defined right ?
Should we handle the other case ?

Reply:
We might want to handle the case where objectName is undefined not to make object name mandatory in the scenario
So i added a check for objectName in `getObjectNameWithBackendFlakyness` to return undefined if it is undefined